website/scripts/ros/ros_host_vehicle_cmd.js

```diff
@@ -22,27 +22,27 @@ function subscribeToVehicleCMD()
         
         if(message!=null && message.ctrl_cmd !=null)
         {
-            //Steering wheel
-            if(message.ctrl_cmd.steering_angle != null)
-            {
-                let rad = message.ctrl_cmd.steering_angle;
-                let rotateDegree = rad % 360; //-35 to +35
-                // console.log('rotateDegree'+rotateDegree);
-                let degreePercent = Math.floor(((rotateDegree/360) * 100));
-                updateSteeringWheel(degreePercent +'%',rotateDegree);
-            }            
+            // //Steering wheel
```

Comment:
Do you still need to leave this commented out?

Reply:
reference only